HW11_1.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+def my_deco (func):
+    def wrap():
```

Comment:
use *args and **kwargs here to be able to decorate a function that takes some arguments

Reply:
done